clouddriver-web/etc/init.d/clouddriver

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Daemon Name: clouddriver
+#
+# chkconfig: 345 91 9
+# description: # description here
+
+# Source function library.
+. /etc/init.d/functions
+
+daemon=clouddriver
+# ** edit the line below - this is the command to start your service **
+prog="/opt/clouddriver/bin/clouddriver"
+lockfile=/var/lock/subsys/$daemon
+PID_FILE="/var/run/${daemon}.pid"
+
+
+start() {
+    #Make some checks for requirements before continuing
+
+    # Start our daemon daemon
+    echo -n $"Starting $daemon ($prog): "
+    MODE=production nohup $prog < /dev/null > /var/log/spinnaker/${daemon}/${daemon}.log 2>&1 &
```

Comment:
oops, that's a typo, it's something that came from a script that started a flask app :)